IpmiFeaturePkg/Include/Policy/IpmiWatchdogPolicy.h

```diff
@@ -0,0 +1,23 @@
+/** @file
+  Definitions for the IPMI watchdog timers policy.
+
+  Copyright (c) Microsoft Corporation
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _IPMI_WATCHDOG_POLICY_H_
+#define _IPMI_WATCHDOG_POLICY_H_
+
+#include <uefi.h>
+
+typedef struct _IPMI_WATCHDOG_POLICY {
+  BOOLEAN    Frb2Enabled;

```

Comment:
With the PCDs being removed, we also lose the comments around what each field means. Could we copy the comments from the PCDs after each policy entry?